app/javascript/dashboard/assets/scss/widgets/_buttons.scss

```diff
@@ -34,7 +34,7 @@
   }
 
   &.hollow {
-    @apply border border-slate-200 dark:border-slate-600 text-woot-700 dark:text-woot-100 hover:bg-woot-50 dark:hover:bg-woot-900;
```

Comment:
Does this break the styles somewhere else?

Reply:
@pranavrajs These changes won't break anything, but some styles were missing I added it in a commit https://github.com/chatwoot/chatwoot/pull/8010/commits/6cbf1210c39a70a23e44354f0860f0ae3499e39a

| Color Scheme  | Light  | Dark  |
| ------------- | ------------- | ------------- |
| Primary  | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/abc87b86-fa62-4c61-8fa7-0fa0e843c589"> | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/b61543e9-d610-4c16-8314-a949b50f6550"> |
| Secondary  | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/1cd18ba8-44b1-4232-a3bc-862355366c4c"> | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/38b41447-ab12-4def-9e3c-ff753a9cdf8c"> |
| Success  | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/ebff5dd9-9f7e-4be4-83b5-5effb06e3639"> | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/37c1e2f9-4dcb-4ae7-ae6c-ebae1da5987b"> |
| Alert  | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/0347edb2-5dba-40c7-964f-c021c8507ef6"> | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/1edc28ff-f926-4d39-b4f4-56e7e5fb7874"> |
| Warning  | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/0a6692f2-5c73-4777-a8e6-d57f1d6a2770"> | <img width="122" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/e6b5e98e-1a5c-4009-9caa-2a4629334d77"> |